server/pulp/server/webservices/views/content.py

```diff
@@ -518,14 +518,36 @@ def get(self, request):
             sources.append(d)
         return generate_json_response_with_pulp_encoder(sources)
 
+
+class ContentSourceCollectionActionView(View):
+    """
+    View for actions on all sources.
+    """
+
+    @staticmethod
+    def refresh(request):
```

Comment:
is there a reason why you made it as a staticmethod and here you left it as it is? https://github.com/jortel/pulp/blob/issue-827/server/pulp/server/webservices/views/content.py#L622